internal/functions/metricsapi/download.go

```diff
@@ -12,33 +12,59 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"net/http"
+	"strings"
 	"time"
 )
 
+var startOfData = time.Date(2020, 10, 23, 0, 0, 0, 0, time.UTC)
+
 //DownloadMetrics Serves most current version of metrics.
 func DownloadMetrics(w http.ResponseWriter, r *http.Request) {
 	var ctx = r.Context()
-	logger := logging.FromContext(ctx).Named("DownloadMetrics")
 
 	client := store.Client{}
 	date := time.Now()
 
+	downloadMetrics(ctx, w, r, client, date)
+}
+
+func downloadMetrics(ctx context.Context, w http.ResponseWriter, r *http.Request, client store.Client, date time.Time) {
```

Comment:
I am not sure if I understand it well. When I want todays metric, I need to provide current date in request?

Reply:
Nope. There's a fallback when no date can be parsed from the request, today's date is used. That's the `date` param provided.

Reply:
@dizider you can but don't have to. Today's date is the default.